src/test/java/org/prgrms/wumo/domain/route/api/RouteControllerTest.java

```diff
@@ -88,6 +96,7 @@ void setup() {
 
 	@AfterEach
 	void tearDown() {
+		routeRepository.deleteById(routeId);
 		partyMemberRepository.deleteById(partyId);
```

Comment:
```suggestion
		partyMemberRepository.deleteById(partyMemberId);
```
운 좋으면 CI 통과하지만 대체로 터지더군요 ㅠㅠ (제 커밋쪽에서 CI 테스트 중에 수정했습니다)